### ISSUE TITLE
Julia installation on Mac does not work

### ISSUE THREAD

**Issue by roiholtzman** [JuliaLang/julia#28254]
I've downloaded Julia 0.6.4 from the official page, and I can't open a Julia terminal on my machine.
I'm running on Mac 10.12.6.

**Comment by stevengj** [comment]
Works for me.  What is the error message?

**Comment by roiholtzman** [comment]
There is no error message. It simply opens and closes immediately.
I have anaconda distribution already installed on my machine. Could this be
the reason?

**Comment by KristofferC** [comment]
What if you execute `/Applications/Julia-0.6.app/Contents/Resources/julia/bin/julia` in the terminal (assuming it is installed there).

**Comment by roiholtzman** [comment]
It runs!! What does it mean?

**Comment by essenciary** [comment]
For 0.6 my recommended approach is to use Homebrew and cask. This will install the Julia app and link it to /usr/local/bin so you'll be able to do `$ julia` in the terminal. 
1. https://brew.sh and follow info to install
2. `$ brew cask install julia`

**Comment by roiholtzman** [comment]
I followed the instructions.
When I type 'Julia' in the terminal, it works. But when I double click on
the Julia icon in the applications folder, it opens and closes.

**Comment by simonbyrne** [comment]
At the terminal, try:
```
osascript << EOF
tell application id "com.apple.terminal"
  do script ("exec '/Applications/Julia-0.6.app/Contents/Resources/julia/bin/julia'")
  activate
end tell
EOF
```

Does that launch Julia in a new window?

**Comment by selasley** [comment]
I'm having the same problem under macos 10.13.6   I can run julia from the command line but double clicking on the application icon only opens a new blank window in the Terminal app.  If I change /Applications/Julia-1.0.app/Contents/Resources/Scripts/main.scpt to

set RootPath to POSIX path of (path to me)
tell application id "com.apple.terminal"
	set win to (do script "")
	do script ("exec '" & RootPath & "Contents/Resources/julia/bin/julia'") in win
end tell

then double clicking the Julia-1.0.app icon launches julia in a new Terminal window

**Comment by simonbyrne** [comment]
Thanks @selasley. Trying to simplify this, could you please try:

```
set RootPath to POSIX path of (path to me)
tell application id "com.apple.terminal"
	set win to do script ("exec '" & RootPath & "Contents/Resources/julia/bin/julia'")
	activate win
end tell
```

**Comment by selasley** [comment]
It turns out the Terminal app was using a copy of bash 4.4.0(1) I compiled as the default shell.  When I changed the "Shells open with" preference back to "Default login shell" or to /bin/tcsh or /bin/zsh double clicking on Julia-1.0.app with the standard main.scpt ran Julia in a new Terminal window as expected.  

I tried the script you suggested.  It worked when Terminal was set to launch the default shell but failed to run Julia when I set the shell to my copy of bash 4.4

So the problem I had appears to be with my copy of bash 4.4 and not with the standard launch script

**Comment by simonbyrne** [comment]
Hmm. I wonder why that happens.

So it turns out you can just specify julia as the shell: I wonder if there is a way to do that from AppleScript?

**Comment by simonbyrne** [comment]
Can you launch Julia from your custom bash?

**Comment by selasley** [comment]
I installed the latest version of bash (4.4.18) with ./configure;make;sudo make install and it has the same behavior - I can run julia from the command line with /Applications/Julia-1.0.app/Contents/Resources/julia/bin/julia  but double clicking the app just opens a new blank Terminal window running the /usr/local/bin/bash shell.    The same applies to Julia 0.6.4 and 0.7.0  None of the ./configure options for bash look like they would change this behavior and I don't really need bash 4.4  so I'll just switch to /bin/bash for the default shell.

I looked around and could not find a way to open a new Terminal window with a specified shell via applescript.

Thanks for your help
